addons/garage/functions/fnc_onAnimationSelect.sqf

```diff
@@ -31,4 +31,5 @@ for "_i" from 0 to (lbSize _ctrlListAnimations - 1) do {
 };
 
 // Update vehicle animations
-[GVAR(center), nil, _animations, true] call BIS_fnc_initVehicle;
+private _vehicle = GVAR(center);
+[QEGVAR(common,initVehicle), [_vehicle, nil, _animations, true], _vehicle] call CBA_fnc_targetEvent;
```

Comment:
For just two uses, I think it would be fine to use the global variable directly.